src/NotInitialisedError.js

```diff
@@ -0,0 +1,3 @@
+export default () => {
+  throw new Error('Initialize i18n with the Provider before calling any internationalization methods');
```

Comment:
PP 

not to throw within the function, but just make NotInitialisedError a specific error and throw it where needed.